### PR TITLE
add delettours page and style it

### PR DESCRIPTION
**Pull request by divinecharlotte**
IN THIS PR I IMPLEMENTED THE FOLLOWING:
- [ ] Added the "Delete item" page with a list of all items with title and a "Delete" button.

### CODE REVIEW

File: src/styles/delete.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+@media (max-width: 680px) {
+  .tour-container {
+    grid-template-columns: 1fr;
+  }
+
+  .delete-container {
+    height: 100%;
+  }
+}
```
Comment (ABDULALI3468) [inline src/styles/delete.css]: Please consider reviewing your styles as they don't seem to be working as expected. For more details I have created a short video below you can look at it.

https://user-images.githubusercontent.com/94223367/197714827-4e68b7b6-fb7b-4935-a972-82b32ed3ed46.mp4

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
### `Few Changes Required`

### Hi @divinecharlotte 

> 

> 
> ## Highlights ✔️
> * [ ]  Linter checks are passing 👏
> * [ ]  Descriptive pull request title and summary 👏> 

> ## Required Changes ♻️
> I have noted a few changes that will improve your project. Please have a look at them and kindly review your project.

- [x] After making all the changes kindly submit the project for review.

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

>

**Comment by divinecharlotte** [comment]
hello  @ABDULALI3468 cause we haven't yet linked the pages to the nav that's why as we are still accessing the through the routes that's why I left it that way and to be fixed after

File: src/styles/delete.css (hunk 1 above)
Comment (ABDULALI3468) [reply src/styles/delete.css]: > hello @ABDULALI3468 cause we haven't yet linked the pages to the nav that's why as we are still accessing the through the routes that's why I left it that way and to be fixed after

@divinecharlotte I suggest you to add the routes even we are not implementing the Navbar for now. But for example if someone is manually going to delete tour page the style will broke. You can implement it by making it `position:fixed; top:0; right:0`. Hope that helps.

**Review by ABDULALI3468 — APPROVED** [review]
Hi @divinecharlotte  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :green_circle: 

- The feature is working as expected.
- PR is descriptive :heavy_check_mark: 
- No Linter error :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------

**Pull request merged**